### DISCUSSION TITLE
GW4 Classic 46mm incorrect screen size

### DISCUSSION THREAD

**Discussion by Knightprozac** [NightscoutFoundation/xDrip#2705] [Q&A]
Edit:  Images added as example.

![Screenshot_20230410_120849_dexdrip](https://user-images.githubusercontent.com/126452399/230852923-57dc0ba4-d97a-4f0e-a7fa-949e313aa0fd.png)

![Screenshot_20230410_154344_dexdrip](https://user-images.githubusercontent.com/126452399/230853308-509a5fd4-bf24-46f5-9f27-20b3e136c111.png)

I have trouble viewing some info on my galaxy watch 4 classic 46mm. It appears that the wrong screen size may be setup as it looks like parts of the display are missing around the edge of the watch screen. This occurs in all sections of the watch xdrip

Details as follows:
Watch model: galaxy watch 4 classic 46mm
Watch model number: sm-r890
Xdrip version: 344deb4-2022.11.16
Phone: galaxy s7
Install method: wear is 3.x method 2

Does anyone know how to fix this?

**Comment by Navid200** [comment]
My hope is that people will use these forums and help.
In the meantime, if you have a facebook account, you can use this group:  https://www.facebook.com/groups/xDripG5

**Reply by Knightprozac** [reply]
Thank you for your suggestion Navid. I'm don't generally use Facebook but I may try it if I can find the time.

If I was less busy I think I'd be able to work up a solution myself but my spare time outside of work is typically occupied by my 1 year old.
